qemu/tests/cgroup.py

```diff
@@ -195,6 +195,7 @@ def param_add_scsi_disks():
                            "add_host", shell=True)
             time.sleep(1)   # Wait for device init
             dev = process.system_output("ls /dev/sd* | tail -n 1", shell=True)
```

Comment:
```suggestion
             dev = process.run("ls /dev/sd* | tail -n 1", shell=True).stdout_text
```

Reply:
@PaulYuuu  Thanks for your review.  seems It doesn't works.  This is test log.
05:38:57 ERROR|   File "/var/lib/avocado/data/avocado-vt/test-providers.d/downloads/io-github-autotest-qemu/qemu/tests/cgroup.py", line 199, in param_add_scsi_disks
    dev = dev.decode()
05:38:57 ERROR| AttributeError: 'str' object has no attribute 'decode'

If comment   #dev = dev.decode(). 
...
stderr: b'/bin/sh: /sys/block/sdc: Is a directory\n/bin/sh: line 1: /queue/rotational: No such file or directory\n'...

Reply:
@xiangchunfu I mean remove the new code and only need to modify the old one.
https://github.com/avocado-framework/avocado/blob/bdf5daad93e8a4f9e542195c7c6ac222d7d32918/avocado/utils/process.py#L350-L355

Reply:
> If comment #dev = dev.decode().
> ...
> stderr: b'/bin/sh: /sys/block/sdc: Is a directory\n/bin/sh: line 1: /queue/rotational: No such file or directory\n'...

Looks like `stdout_text` will include `\n`, so `process.getoutput("ls /dev/sd* | tail -n 1", shell=True)` or `process.run("ls /dev/sd* | tail -n 1", shell=True).stdout_text.strip()` can fix it.

Reply:
@PaulYuuu  Thanks.  It works.  updated